src/test/java/org/junit/internal/runners/statements/FailOnTimeoutTest.java

```diff
@@ -37,122 +34,176 @@
  */
 @RunWith(Parameterized.class)
 public class FailOnTimeoutTest {
+    private static final long TIMEOUT = 100;
```

Comment:
These changes appear to come from 2db6394, but show up as differences, which makes it hard to review.